src/modules/map/components/cesium/CesiumMouseTracker.vue

```diff
@@ -0,0 +1,121 @@
+<script setup>
+import { Cartographic, Math, ScreenSpaceEventHandler, ScreenSpaceEventType } from 'cesium'
+import { transform } from 'ol/proj'
+import { computed, inject, nextTick, onMounted, onUnmounted, ref, watch } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useStore } from 'vuex'
+
+import allFormats, { LV03Format, LV95Format } from '@/utils/coordinates/coordinateFormat'
+import log from '@/utils/logging'
+
+const mousePosition = ref(null)
+const displayedFormatId = ref(LV95Format.id)
+
+const store = useStore()
+const i18n = useI18n()
+
+const is3DReady = computed(() => store.state.cesium.isViewerReady)
+const projection = computed(() => store.state.position.projection)
+
+const dispatcher = { dispatcher: 'CesiumMouseTracker.vue' }
+const getViewer = inject('getViewer', () => {}, true)
+
+let handler = null
+
+watch(
+    is3DReady,
+    (newValue) => {
+        if (newValue) {
+            setupHandler()
+        }
+    },
+    { immediate: true }
+)
+
+onMounted(() => {
+    nextTick(() => {
+        setupHandler()
+    })
+})
+
+onUnmounted(() => {
+    if (handler) {
+        handler.removeInputAction(ScreenSpaceEventType.MOUSE_MOVE)
+    }
+})
+
+function setupHandler() {
+    if (!getViewer()) {
+        return
+    }
+    handler = new ScreenSpaceEventHandler(getViewer().scene.canvas)
+    const viewer = getViewer()
+    handler.setInputAction((movement) => {
+        const ray = viewer.camera.getPickRay(movement.endPosition)
+        const cartesian = viewer.scene.globe.pick(ray, viewer.scene)
+        if (cartesian) {
+            const cartographic = Cartographic.fromCartesian(cartesian)
+            const longitude = Math.toDegrees(cartographic.longitude)
+            const latitude = Math.toDegrees(cartographic.latitude)
+            let coordinate = transform([longitude, latitude], 'EPSG:4326', projection.value)
+            coordinate.push(cartographic.height)
+
+            mousePosition.value.textContent = formatCoordinate(coordinate)
+        }
+    }, ScreenSpaceEventType.MOUSE_MOVE)
+}
+
+function showCoordinateLabel(displayedFormat) {
+    return displayedFormat?.id === LV95Format.id || displayedFormat?.id === LV03Format.id
+}
+
+function setDisplayedFormatWithId() {
+    store.dispatch('setDisplayedFormatId', {
+        displayedFormatId: displayedFormatId.value,
+        ...dispatcher,
+    })
+}
+
+function formatCoordinate(coordinate) {
+    const displayedFormat = allFormats.find((format) => format.id === displayedFormatId.value)
+    if (displayedFormat) {
+        if (showCoordinateLabel(displayedFormat)) {
+            return `${i18n.t('coordinates_label')} ${displayedFormat.format(
+                coordinate,
+                projection.value
+            )}`
+        }
+        return displayedFormat.format(coordinate, projection.value, true)
+    } else {
+        log.error('Unknown coordinates display format', displayedFormatId.value)
+    }
+}
```

Comment:
There's already a helper to do that for you, you can have a look at `src/utils/coordinates/coordinateFormat.js`, each format gives you a coordinate to string function